renderer/d3d/pls_render_context_d3d_impl.cpp

```diff
@@ -716,8 +716,13 @@ ID3D11RenderTargetView* PLSRenderTargetD3D::targetRTV()
 {
     if (m_targetRTV == nullptr && m_targetTexture != nullptr)
     {
+        D3D11_RENDER_TARGET_VIEW_DESC desc;
+        ZeroMemory(&desc, sizeof(desc));
+        desc.Format = DXGI_FORMAT_R8G8B8A8_UNORM;
```

Comment:
Can you set desc.Format to `m_targetFormat`?

Reply:
`m_targetFormat` will be `R8G8B8A8_TYPELESS` in this case, which isn't a valid type for a view since it needs to be fully qualified. Maybe we could do a switch statement on the format and pick the appropriate type for the view?

Reply:
I see. Let's switch it then and convert the typeless format to UNORM.